openupgradelib/openupgrade_tools.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
This will work in python2 context ?

Reply:
Well, I think there aren't Unicode characters, but I'm looking how to preserve that, as black removes it.

Reply:
I think you need to pin black to 21.12b0, which is the last version that supports python 2 code.

Reply:
Tried, and it's still removing it. I'm digging more now.

Reply:
Found. It's this one: https://github.com/OCA/openupgradelib/pull/317/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R71-R72

Removing it, it's not done.